protos/monitoring.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+package proto;
+
+option java_package = "fi.hsl.common.monitoring.proto";
+option java_outer_classname = "Monitoring";
+
+message MonitoringMessage {
+    required int32 SchemaVersion = 1 [default = 1];
+    optional int64 timestamp = 2;
+    required string key = 3;
+    required double value = 4;
```

Comment:
Are we sure it's always a double? Counts are usually better as integers.. 

Can be done by having optional fields for both and perhaps a type-enumeration.. Do we wanna go that way? Of course we can always create a schema version 2.

Reply:
The idea is that the value is always numeric so `double` could represent it whether it is `int` or `float` (and small errors are not a problem?).